changelog/5985.improvement.md

```diff
@@ -0,0 +1,7 @@
+Implemented model checkpointing for DIET (including the response selector) and TED. The best models during training will be stored instead of just the last model. The model is evaluated on the basis of `evaluate_every_number_of_epochs` and `evaluation_on_number_of_examples`.
```

Comment:
```suggestion
Implemented model checkpointing for DIET (including the response selector) and TED. The best model during training will be stored instead of just the last model. The model is evaluated on the basis of `evaluate_every_number_of_epochs` and `evaluation_on_number_of_examples`.

```